### PR TITLE
5578 Update "how to build your name" and "helpful tips" components

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#5578

*Description of changes:*
 - Update "how to build your name" and "helpful tips" components
 - Width changes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/components/lower-info-area/lower-container.vue [hunk 1]
```diff
@@ -76,37 +82,39 @@
...
+          </v-container>
+        </v-col>
+        <v-col cols="6">
+          <v-container class="pa-9">
+            <v-row alinn="start" align-content="start" align-items="start">
+              <v-col cols="12" class="h3 info-underline px-0">Contact Us</v-col>
+              <v-col cols="12" class="px-0">
+                For support or questions about this application contact us at:
+              </v-col>
+              <ul class="contact-info-list">
```
Comment (severinbeauvais) [inline client/src/components/lower-info-area/lower-container.vue:106]: NB to all devs: If we come across another use of this contact info list, let's pull it out into a separate component and reuse this code.

**Review by severinbeauvais — APPROVED** [review]
Good use of helper classes and CSS variables. Nice job!

File: client/src/components/lower-info-area/lower-container.vue [hunk 2]
```diff
@@ -125,43 +133,50 @@ export default class LowerContainer extends Vue {
...
 
-<style lang="sass" scoped>
-@import "@/assets/scss/theme.scss"
+<style lang="scss" scoped>
+@import "@/assets/scss/theme.scss";
 
-li
-  font-size: 16px
+li {
+  font-size: 16px;
```
Comment (cameron-freshworks) [inline client/src/components/lower-info-area/lower-container.vue:140]: Did you want to update to `1rem`?

File: client/src/components/lower-info-area/lower-container.vue [hunk 3]
```diff
@@ -125,43 +133,50 @@ export default class LowerContainer extends Vue {
...
-#lower-container
-  padding: 0
-  margin: 0
-  font-size: 16px
-  color: $text
-  background-color: #F1F3F5
+#lower-container {
+  padding: 0;
+  margin: 0;
+  font-size: 16px;
```
Comment (cameron-freshworks) [inline client/src/components/lower-info-area/lower-container.vue:146]: Same as above

File: client/src/components/lower-info-area/lower-container.vue [hunk 4]
```diff
@@ -125,43 +133,50 @@ export default class LowerContainer extends Vue {
...
 
-.square-card
-  height: 260px
-  width: 260px
-  padding: 15px 30px 30px 15px
-  border-radius: 4px
-  background-color: white
-  box-shadow: 2px 2px 4px 0 darkgray
+    .launch-icon {
+      font-size: 16px;
```
Comment (cameron-freshworks) [inline client/src/components/lower-info-area/lower-container.vue:176]: Same as above, though i'm not 100% sure if the icon size translates the same to rem. Worth a look!

Comment (severinbeauvais) [reply client/src/components/lower-info-area/lower-container.vue:176]: Usually we can get away with "small" or "large" (or neither) icon property for desired size.

File: client/src/components/lower-info-area/lower-container.vue [hunk 5]
```diff
@@ -125,43 +133,50 @@ export default class LowerContainer extends Vue {
...
+      color: $link;
+      padding-left: 4px;
```
Comment (cameron-freshworks) [inline client/src/components/lower-info-area/lower-container.vue:178]: No action required but Vuetify helper class equivalent is `pl-1`. I'm pretty sure each number is the equivalent of 4px, i think*

File: client/src/components/lower-info-area/name-build-sub-components/example-name.vue [hunk 6]
```diff
@@ -67,30 +53,26 @@ import { Component, Vue } from 'vue-property-decorator'
...
+<style scoped lang="scss">
+@import "@/assets/scss/base.scss";
 
-.text-divider-1
-  margin-left: 6.5rem
-  border-left: 1px solid rgba(0, 0, 0, 0.12)
-  height: 2rem
+#example-name {
+  p {
+    font-size: 14px;
```
Comment (cameron-freshworks) [inline client/src/components/lower-info-area/name-build-sub-components/example-name.vue:61]: Same as above but `.875rem`. 
I seem really focused on font sizes here 😆

**Review by cameron-freshworks — APPROVED** [review]
Looks great! Just a few small comments.

File: client/src/components/lower-info-area/name-build-sub-components/example-name.vue (hunk 6 above)
Comment (severinbeauvais) [reply client/src/components/lower-info-area/name-build-sub-components/example-name.vue:61]: You are correct. I missed these :)

**Pull request merged**